packages/core/src/codewhisperer/commands/basicCommands.ts

```diff
@@ -183,23 +184,13 @@ export const connectWithCustomization = Commands.declare(
             customizationArn?: string,
             customizationNamePrefix?: string
         ) => {
+            SsoAccessTokenProvider.authSource = source
```

Comment:
future idea: could we not grab the current metric here, and annotate it with telemetry.record()? that could avoid the `static source`

Reply:
> is this not during a telemetry.run() context

Bingo. We _could_ wrap the following calls in a telemetry.run() event, and the other changes would detect this and attach the metrics to the existing run event. But, this code path may not result in a browser auth event in certain circumstances. It would be wrong to emit the metric in those cases.